picking_good_slis.md

```diff
@@ -0,0 +1,86 @@
+# Picking Good SLIs
+
+## SRE's Philosophy on Monitoring
+
+A core goal of Site Reliability Engineering (SRE) is to ensure the availability of a service. "Availability" here means much more than simply "is my application running." Effectively monitoring a modern cloud application requires insight into the application's response time, error rate, traffic level, and more.
+
+Furthermore, the scope of monitoring should not be limited solely for use in deciding when to page an operations team. Metrics should be closely tied to business objectives, informing decisions on what features to prioritize and when to focus on adding new features versus improving reliability.
```

Comment:
```suggestion
Furthermore, the scope of monitoring should not be limited solely for use in deciding when to page an operations team. Metrics should be closely tied to business objectives, informing decisions on what work to prioritize and when to focus on adding new features versus improving reliability.
```

Do you mean "work" here and not "features?"

Reply:
Fixed.